labelbox/schema/model_run.py

```diff
@@ -80,27 +80,40 @@ def upsert_labels(self, label_ids, timeout_seconds=3600):
             }})['MEALabelRegistrationTaskStatus'],
                                      timeout_seconds=timeout_seconds)
 
-    def upsert_data_rows(self, data_row_ids, timeout_seconds=3600):
+    def upsert_data_rows(self,
+                         data_row_ids=None,
+                         global_keys=None,
+                         timeout_seconds=3600):
         """ Adds data rows to a Model Run without any associated labels
         Args:
-            data_row_ids (list): data row ids to add to mea
+            data_row_ids (list): data row ids to add to model run
+            global_keys (list): global keys for data rows to add to model run
             timeout_seconds (float): Max waiting time, in seconds.
         Returns:
             ID of newly generated async task
         """
+        if data_row_ids is None == global_keys is None:
+            raise ValueError(
+                "Must provide either data_row_ids or global_keys, but not both."
+            )
 
-        if len(data_row_ids) < 1:
+        if data_row_ids is not None and len(data_row_ids) < 1:
             raise ValueError("Must provide at least one data row id")
 
+        if global_keys is not None and len(global_keys) < 1:
+            raise ValueError("Must provide at least one global key")
+
         mutation_name = 'createMEAModelRunDataRowRegistrationTask'
-        create_task_query_str = """mutation createMEAModelRunDataRowRegistrationTaskPyApi($modelRunId: ID!, $dataRowIds : [ID!]!) {
-          %s(where : { id : $modelRunId}, data : {dataRowIds: $dataRowIds})}
+        create_task_query_str = """mutation createMEAModelRunDataRowRegistrationTaskPyApi($modelRunId: ID!, $dataRowIds: [ID!], $globalKeys: [ID!]) {
```

Comment:
Is this a breaking change for customers who are on prev versions of SDK that still have `$dataRowIds: [ID!]!`, if the backend is now accepting `[ID!]`? Might be worth testing

Reply:
I tested this by passing dataRowIds and not globalKeys in the GraphQL call.

Reply:
Also ran `tests/integration/annotation_import/test_model_run.py::test_model_run_upsert_data_rows` in develop branch to test backwards compatibility of the API.